packages/common-types/src/transactions.ts

```diff
@@ -395,4 +423,5 @@ export enum TransactionApplicationIdentifier {
   TRANSFER = 'TRANSFER',
   SWAP = 'SWAP',
   DCA = 'DCA',
+  EARN_DEPOSIT = 'EARN_DEPOSIT',
```

Comment:
Yeah we should actually have both here since we use them for the satisfaction

Reply:
As for the naming, the method on the SC will always be called deposit for a first time deposit so kinda following that pattern, since the function on the earnService is also going to be called deposit

Reply:
Okay, but i'm seeing that we are using `deposit` for both terms: deposit (create + increase) and just create. I think we should not use deposit term when we want to differentiate create and increase (in transaction history for example), just to prevent asuming that the 'deposit' name is englobing all